src/Models/Cities.php

```diff
@@ -0,0 +1,23 @@
+<?php
+declare(strict_types=1);
+
+namespace Canvas\Models;
+
+class Cities extends AbstractModel
```

Comment:
@FredPeal lets create a folder insides models name Geo Or Location and add this models there to start structuring things out